entries/b/bokio.se.json

```
@@ -0,0 +1,20 @@
+{
+  "Bokio Sverige": {
+    "domain": "bokio.se",
+    "url": "https://www.bokio.se",
+    "img": "bokio.svg",
+    "tfa": [
+      "totp",
+      "custom-software"
+    ],
+    "notes": "Custom software BankID used for users using built-in banking features",
+    "documentation": "https://www.bokio.se/hjalp/bokio-medlemskap/ditt-bokiokonto/tvafaktor-autentisering-med-bokio/",
+    "keywords": [
+      "finance",
+      "banking"
```

The banking category is meant for companies that are actual banks. For accounting software the finance category alone can be used.
```suggestion
      "finance"
```